soh/src/overlays/actors/ovl_En_Hy/z_en_hy.c

```diff
@@ -599,67 +599,67 @@ s16 func_80A70058(PlayState* play, Actor* thisx) {
                     Player_UpdateBottleHeld(play, GET_PLAYER(play), ITEM_BOTTLE, PLAYER_IA_BOTTLE);
                     break;
                 case 0x7016:
-                    gSaveContext.infTable[12] |= 1;
+                    Flags_GetInfTable(INFTABLE_C0);
```

Comment:
not sure what happened here, but `|= 1`, but there are a bunch of `|=` being turned into get when they should be set

Reply:
Good catch, I'll tweak the script and re-run

Reply:
<img width="143" alt="Screenshot 2023-06-16 at 11 12 13 PM" src="https://github.com/HarbourMasters/Shipwright/assets/7316699/1b6f28f1-78ae-46cd-b3cd-0e91f556a3b3">

lol well that will do it

Reply:
Fixed 👍